modules/azure-firewall/outbound.tf

```diff
@@ -0,0 +1,15 @@
+data "azurerm_route_table" "default" {
+  count               = "${length(var.aks)}"
+  name                = "${lookup(var.aks[count.index], "rt_name")}"
+  resource_group_name = "${lookup(var.aks[count.index], "rg_name")}"
+}
+
+resource "azurerm_route" "default" {
+  count                  = length(var.firewall)
+  name                   = "defaultOutbound"
```

Comment:
check naming conventions for route tables, for AKS we had -route-table not sure if there's a standard for it
https://portal.azure.com/#blade/HubsExtension/BrowseResourceBlade/resourceType/Microsoft.Network%2FrouteTables